docs/usage/arrays.md

```diff
@@ -290,4 +290,41 @@ const uiSchema = {
 render((
   <Form schema={schema} uiSchema={uiSchema} />
 ), document.getElementById("app"));
+```
+
+## Array-group
+
+The default behavior for array group is a tab component. It groups an array of items
+
+Example:
+
+```jsx
+const schema = {
+  type: "array",
+  title: "Member Info",
+  group: [
```

Comment:
Consider making this a `uiSchema` option instead of a non-standard `JSONSchema` option